<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>constellationWizardDialog</class>
 <widget class="QDialog" name="constellationWizardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Constellation Wizard</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>420</y>
     <width>129</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>420</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="constellationParametersGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>405</height>
    </rect>
   </property>
   <property name="title">
    <string>Constellation Parameters</string>
   </property>
   <widget class="QComboBox" name="constTypeComboBox">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>20</y>
      <width>218</width>
      <height>26</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Walker Constellation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flower Constellation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Self-defined Constellation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Along-track Formation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cross-track Formation</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>24</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="constellationParametersStackWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>397</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="WalkerPage">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>20</y>
        <width>381</width>
        <height>169</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="inclinationLabel">
         <property name="text">
          <string>Inclination [deg]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="inclinationLineEdit">
         <property name="text">
          <string>56</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="numberOfOrbitPlanesLabel">
         <property name="text">
          <string>Number of orbit planes</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numberOfOrbitPlanesLineEdit">
         <property name="text">
          <string>3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="numberOfSatellitesLabel">
         <property name="text">
          <string>Number of satellites per orbit plane</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="numberOfSatellitesLineEdit">
         <property name="text">
          <string>9</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="semiMajoraxisLabel">
         <property name="text">
          <string>Semimajor Axis [km]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="semiMajoraxisLineEdit">
         <property name="text">
          <string>29600</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="relSpacingLabel">
         <property name="toolTip">
          <string>natural number between 0 and number of orbit planes -1</string>
         </property>
         <property name="text">
          <string>Relative spacing between sat. in adjacent planes</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="relSpacingLineEdit">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="FlowerPage">
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>376</width>
        <height>308</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="numberOfPetalsLabel">
         <property name="text">
          <string>Number of petals</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numberOfPetalsLineEdit">
         <property name="text">
          <string>3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="numberOfDaysToRepeatGroundTrackLabel">
         <property name="text">
          <string>Number of days to repeat ground track</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="numberOfDaysToRepeatGroundTrackLineEdit">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="numberOfSatellitesLabel_2">
         <property name="text">
          <string>Number of satellites</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="numberOfSatellitesLineEdit_2">
         <property name="text">
          <string>12</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="perigeeLabel">
         <property name="text">
          <string>Argument of periapsis [deg]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="perigeeLineEdit">
         <property name="text">
          <string>270.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="inclinationLabel_2">
         <property name="text">
          <string>Inclination [deg]</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="inclinationComboBox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <item>
          <property name="text">
           <string>63.4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>116.6</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="heightOfPerigeeLabel">
         <property name="text">
          <string>Height of periapsis [km]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="heightOfPerigeeLineEdit">
         <property name="text">
          <string>7000.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="schemeLabel">
         <property name="text">
          <string>scheme</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="schemeComboBox">
         <item>
          <property name="text">
           <string>symmetric</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>asymmetric</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="raanOfFirstSatelliteLabel">
         <property name="text">
          <string>Raan of first satellite</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="raanOfFirstSatelliteLineEdit">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="initialTrueAnomalyOfFirstSatelliteLabel">
         <property name="text">
          <string>Initial true anomaly of first satellite</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="initialTrueAnomalyOfFirstSatelliteLineEdit">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="limitedOrbitNodeRangeLabel">
         <property name="text">
          <string>Limited orbit node range</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="limitedOrbitNodeRangeLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>45.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="SelfDefinedPage">
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>20</y>
        <width>297</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="numberOfSatellitesLabel_3">
         <property name="text">
          <string>Number of Satellites</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numberOfSatellitesLineEdit_3">
         <property name="text">
          <string>5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Seite"/>
    <widget class="QWidget" name="Seite_2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>constellationWizardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>constellationWizardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>constellationParametersStackWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
